graph/pkg/service/v0/service.go

```diff
@@ -101,3 +108,19 @@ func NewService(opts ...Option) Service {
 
 	return svc
 }
+
+// parseHeaderPurge parses the 'Purge' header.
+// Expected values are 'T' or 'F' case insensitive.
+// If the header is not set or if any other values
+// are present in the header false is returned.
+func parsePurgeHeader(h http.Header) bool {
+	val := h.Get(HeaderPurge)
+	switch strings.TrimSpace(val) {
+	case "t", "T":
```

Comment:
Maybe it makes sense to just use `ParseBool` function? https://golang.google.cn/pkg/strconv/#ParseBool

Imho it would be much clearer which values are supported...

Reply:
Yeah, we could use ParseBool here, too. @butonic and I took the idea from the webdav spec. There are headers like `Overwrite: [T|F]`. I don't know if they support 0,1 though.

Reply:
Ah I see. I'd say it's cleaner if we do so, but if you already thought about it and decided not to use it it is fine for me too...

Reply:
I just changed it to `ParseBool`. Shouldn't hurt I guess and is cleaner.